mountinfo/mounted_linux.go

```diff
@@ -7,6 +7,49 @@ import (
 	"golang.org/x/sys/unix"
 )
 
+// MountedFast is a method of detecting mount points without reading
+// mountinfo from procfs. A mount-point check is guaranteed to be a
+// mount-point or not only when sure is true. When sure is false, the
```

Comment:
nit: `s/mount-point/mount point/`. Here "mount point" is a common term, see e.g. man 8 mount.